### PR TITLE
[WIP][8.0][website_multi_image] Provide reusable templates for base_multi_image.

### PR DESCRIPTION
**Pull request by yajo**
WIP until https://github.com/OCA/server-tools/pull/374 is merged.

This just serves as a base for other modules based on `website` and `base_multi_image`. Provides some reusable templates and styling.

@rafaelbn

### CODE REVIEW

File: website_multi_image/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+It will be automatically installed if any other module you have needs those
+tools, but otherwise you probably don not need it.
+
+Development
+===========
+
+To develop a module based on this one:
+
+* See module ``website_event_multi_image`` as an example.
+* Use the provided templates in frontend website views.
```
Comment (pedrobaeza) [inline website_multi_image/README.rst]: I think this sentence should be rewritten to:

```
Inherit from the provided QWeb templates for creating website views.
```

And then you can list the current views IDs with a little explanation of each.

File: website_multi_image/static/src/css/website_multi_image.sass [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@charset "utf-8"
+/* © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+#multi_image_carousel
```
Comment (pedrobaeza) [inline website_multi_image/static/src/css/website_multi_image.sass:5]: Is this commented or is the notation for sass files?

**Comment by pedrobaeza** [comment]
For the icon, you can overlap the current website icon over the base image.

File: website_multi_image/static/src/css/website_multi_image.sass (hunk 2 above)
Comment (yajo) [inline website_multi_image/static/src/css/website_multi_image.sass:5]: Bad highlighter.

**Comment by yajo** [comment]
OCA/server-tools#374 is merged. WIP no more please, and add needs review please.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by andhit-r** [comment]
:+1:

**Pull request merged**